src/Entity/AlertBannerEntity.php

```diff
@@ -406,7 +409,21 @@ public function getCacheContexts() {
       $this->addCacheContexts($contexts);
```

Comment:
As we're setting cache contexts outside the visibility field check I think this line should be moved out of the check block. Although this might not be needed at all.

Reply:
> Although this might not be needed at all.

Seem to be correct.